app/src/components/PlayersGroup.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { Image, StyleSheet, Text, View } from 'react-native';
+
+import { TextStyle, theme } from '../theme';
+import MyAvatar from './shared/MyAvatar';
+
+interface PlayersGroupProps {
+  firstTeamAvatarList: string[];
+  firstTeamName: string;
+  secondTeamAvatarList: string[];
+  secondTeamName: string;
+  notReadyPlayersAvatarList?: string[];
+}
+
+export const PlayersGroup: React.FC<PlayersGroupProps> = (props) => {
+  return (
+    <View style={{ flex: 1 }}>
+      <View style={styles.label}>
+        <Text style={TextStyle.noteH1}>Zespół {props.firstTeamName}</Text>
+      </View>
+      <View style={styles.avatarsList}>
+        {props.firstTeamAvatarList.map((imgSrc, index) => (
+          <MyAvatar height={50} width={50} src={imgSrc} />
```

Comment:
Mapowane elementy powinny mieć prop `key` (https://reactjs.org/docs/lists-and-keys.html)